ROMFS/px4fmu_common/init.d/4100_tiltquadrotor

```diff
@@ -0,0 +1,35 @@
+#!nsh
+#
+# @name Tilt-Quadrotor
+#
+# @url http://www.alivaero.com/the-project.html
+#
+# @type Tilt-Quad
```

Comment:
I believe it's "VTOL Tiltrotor" for type and "VTOL" for class.

https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/13012_convergence#L5-L6